api/app/GraphQL/Validators/UpdatePublishedPoolInputValidator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\GraphQL\Validators;
+
+use App\Enums\PoolStatus;
+use App\Models\Pool;
+use Database\Helpers\ApiErrorEnums;
+use Illuminate\Validation\Rule;
+use Nuwave\Lighthouse\Validation\Validator;
+
+final class UpdatePublishedPoolInputValidator extends Validator
+{
+    /**
+     * Return the validation rules.
+     *
+     * @return array<string, array<mixed>>
+     */
+    public function rules(): array
+    {
+
+        $pool = Pool::find($this->arg('id'));
+
+        return [
+            'changeJustification' => [
+                Rule::when($pool->status === PoolStatus::PUBLISHED->name, [
+                    'required',
+                ]),
+            ],
```

Comment:
I guess we could just make this always required? :thinking: That would remove the need for querying the pool.